_config.yml

```diff
@@ -289,11 +289,19 @@ picture:
   nomarkdown: false
   suppress_warnings: true
   source: ""
-  output: "assets/images"
+  output: "../generated"
```

Comment:
Does that generate the images in the repo directory rather than the built site directory?

Reply:
Generated images are placed in the `generated` folder to mitigate against issues with `jekyll-polyglot` & `jekyll_picture_tag` clashing.